comp110-worksheet-6/OxoBoard.cs

```diff
@@ -10,42 +10,125 @@ public enum Mark { None, O, X };
 
 	public class OxoBoard
 	{
-		// Constructor. Perform any necessary data initialisation here.
-		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+		Mark[,] boardSpaces = new Mark[3,3]; //Creates a new array using the Mark enumerable
+		public int fullSpaces; //creates an int value representing the filled spaces
+		List<Mark> occupiedSpacesX = new List<Mark>(); //Two seperate lists made to contain the used spaces for X and O
+		List<Mark> occupiedSpacesO = new List<Mark>();
+
+		public OxoBoard()
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+			boardSpaces.Initialize();
 		}
 
 		// Return the contents of the specified square.
 		public Mark GetSquare(int x, int y)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+			if (Mark.X.Equals(true))
```

Comment:
This doesn't do what you think it does -- you are checking if `Mark.X` (which is an enum value) is equal to the boolean value `true`. If you are trying to check the contents of a grid square then you need to access it as `boardSpaces[x, y]`